packages/frontend/tailwind.config.ts

```diff
@@ -50,6 +51,11 @@ const config = {
           DEFAULT: 'hsl(var(--card))',
           foreground: 'hsl(var(--card-foreground))',
         },
+        main: {
+          green: '#01C38D',
+          darkGreen:'#008E67',
+          darkBlue:'#031C30'
```

Comment:
Move these color definitions into global.css
`--green: #01C38D` ...

that way, we will be able to use the var(--green) on multiple components from within the tailwind configuration

For example, our primary color is `main-green` and secondary color is `main-dark-blue` ( or main-light-blue, need to check with @WouroudElKhaldi )
defining those colors as vars in the global.css will allow u to style the colors of different components from within the tailwind.config by using var(--primary) instead of going into each component and mentioning that its color is main-green